### PR TITLE
[orchagent] Arm 32-bit arch compilation warning Fixes

### PR DESCRIPTION
**Pull request by Sabareesh-Kumar-Anandan**
Signed-off-by: Sabareesh Kumar Anandan <sanandan@marvell.com>

**What I did**
Replaced %l variants with PRI format specifier to fix compilation warnings for arm 32-bit arch

**Why I did it**

**How I verified it**
Compiled and verified for arm 32-bit arch (marvell-armhf)

**Details if related**

### CODE REVIEW

File: orchagent/mirrororch.cpp [hunk 1]
```diff
@@ -748,7 +748,7 @@ bool MirrorOrch::setUnsetPortMirror(Port port,
     if (set)
     {
         port_attr.value.objlist.count = 1;
-        port_attr.value.objlist.list = (long unsigned int *)calloc(port_attr.value.objlist.count, sizeof(sai_object_id_t));
+        port_attr.value.objlist.list = reinterpret_cast<sai_object_id_t *>((void *)calloc(port_attr.value.objlist.count, sizeof(sai_object_id_t)));
```
Comment (qiluo-msft) [inline orchagent/mirrororch.cpp:751]: >calloc [](start = 83, length = 6)

`calloc` already returns a `void *`. Why you need to convert explicitly?

**Review by qiluo-msft — CHANGES_REQUESTED** [review]
As comment

File: orchagent/mirrororch.cpp (hunk 1 above)
Comment (Sabareesh-Kumar-Anandan) [reply orchagent/mirrororch.cpp:751]: reinterpret cast reports error if we don't explicitly type cast

Comment (qiluo-msft) [reply orchagent/mirrororch.cpp:751]: I mean why convert `calloc` to `void *`. It is already `void *`.

Comment (Sabareesh-Kumar-Anandan) [reply orchagent/mirrororch.cpp]: removed explicit cast

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**